Services/DeviceModels.cs

```diff
@@ -143,6 +149,35 @@ public class DeviceModels : IDeviceModels
             await this.customDeviceModels.DeleteAsync(id);
         }
 
+        /// <summary>
+        /// Get property names from all device models.
+        /// </summary>
+        public async Task<List<DeviceProperty>> GetPropertyNamesAsync()
+        {
+            /*
+             * This will only return the device model properties,
+             * this will not return all of the properties that are actually
+             * associated with the devices after start-up.
+             * The properties added to devices after startup will come from the
+             * new API located in IOT Hub Manager.
```

Comment:
We can remove this comment. This is returning all props for device model. Additional comment for device props is not needed here. Device simulation does not use IoTHub Manager, no need to add a comment here.

Reply:
done